src/CodeGen_LLVM.cpp

```diff
@@ -2231,6 +2231,25 @@ void CodeGen_LLVM::visit(const Call *op) {
     } else if (op->is_intrinsic(Call::interleave_vectors)) {
         internal_assert(0 < op->args.size());
         value = interleave_vectors(op->type, op->args);
+    } else if (op->is_intrinsic(Call::slice_vector)) {
+        const IntImm *num_vector_args = op->args[0].as<IntImm>();
```

Comment:
The number of args should just be op->args.size() - 2, not an explicit argument. To make things a little less wonky, you could make it so the first 2 arguments are the start and size, and then the remaining arguments are vectors to be concatenated. Or, we should just add the concat_vectors intrinsic now, and use that (so slice_vector would always take 3 arguments).